tests/utils.py

```diff
@@ -17,20 +17,17 @@
 from TileStache import getTile, parseConfig
 from TileStache.Core import KnownUnknown
 
-def request(config_file_content, layer_name, format, row, column, zoom):
+def request(config_content, layer_name, format, row, column, zoom):
```

Comment:
I'm seeing that you are sending an string rather than a dictionary. Also, fix the documentation.